src/raftstore/coprocessor/dispatcher.rs

```diff
@@ -143,19 +148,22 @@ mod test {
         bypass_post: Arc<RwLock<bool>>,
         called_pre: Arc<RwLock<u8>>,
         called_post: Arc<RwLock<u8>>,
+        return_err: Arc<RwLock<bool>>,
     }
 
     impl TestCoprocessor {
         fn new(bypass_pre: Arc<RwLock<bool>>,
                bypass_post: Arc<RwLock<bool>>,
                called_pre: Arc<RwLock<u8>>,
-               called_post: Arc<RwLock<u8>>)
+               called_post: Arc<RwLock<u8>>,
+               return_err: Arc<RwLock<bool>>)
```

Comment:
No, we use this argument to emulate a condition that return a error in the middle running.